app/scripts/directives/logViewer.js

```diff
@@ -197,7 +194,8 @@ angular.module('openshiftConsole')
               var fill = $scope.fixedHeight ? $scope.fixedHeight : Math.floor($(window).height() - contentTop - pulserHeight);
               if (!$scope.chromeless && !$scope.fixedHeight) {
                 // Add some bottom margin if not chromeless.
```

Comment:
Yeah, I can't find a cleaner way. I was thinking of trying to get the padding/margin on the element in JS, but the element with the margin keeps changing.